### DISCUSSION TITLE
Cant use my credit card

### DISCUSSION THREAD

**Discussion by talist225** [planetscale/discussion#529] [FAQs]
Hey,
I want to create a new database but i cant use my credit card again for free plan. The problem here is i dont rembmer the email and password i used for this test user i made. i want to cancel it, how can i trace it? and why cant i use my own card again for another free database in new user?

**Comment by orware** [comment]
Hello @talist225,

Thank you for your interest in using PlanetScale. 

As we offer a generous free tier, we now require a debit or credit card on file to limit potential abuse of the platform. 

I believe we have touched base together on your tickets that were submitted separately from your message here earlier today.

**Reply by aviral17** [reply]
I can't use my Debit Card, Discover or Visa or MasterCard all are declined, why man ??? I want to use Free Tier. Also last month I created project with Prisma PlanetScale, that time they haven't asked for any card, and I successfully used free tier, but now I can't even see my Database/Project which I created last month,  All vanished,  I mean seriously ??? Why you doing this ??? Please help me
